Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ColorPicker/ColorPickerButtonXaml.bind

```diff
@@ -131,5 +131,43 @@
         </controls:ColorPickerButton.ColorPickerStyle>
       </controls:ColorPickerButton>
     </StackPanel>
+    <!-- Example 5 -->
+    <StackPanel Grid.Row="3"
```

Comment:
The new example is sitting on top of an existing one, that's not what was intended?
```suggestion
    <StackPanel Grid.Row="4"
```

Reply:
Good catch, I'll update

Reply:
It was originally a grid when we had examples stacked both horizontally and vertically. It was changed to just vertical later on so StackPanel makes more sense. Its what I was expecting and why copy/paste got me. I'll switch to a StackPanel the same as ColorPicker.

Reply:
Updated to fix overlap and use a `StackPanel` instead. `ColorPicker` and `ColorPickerButton` samples are now almost the same XAML structure.